master/internal/api_checkpoint.go

```diff
@@ -28,6 +37,59 @@ func (a *apiServer) GetCheckpoint(
 	}
 }
 
+func (a *apiServer) DeleteCheckpoints(
+	ctx context.Context,
+	req *apiv1.DeleteCheckpointsRequest) (*apiv1.DeleteCheckpointsResponse, error) {
+	deleteCheckpoints := req.CheckpointUuids
+	dCheckpointsInModelRegistry, _ := a.m.db.GetDeleteCheckpointsInModelRegistry(deleteCheckpoints)
```

Comment:
we absolutely have to handle errors. also I think this name is much clearer
```suggestion
	registeredCheckpointUUIDs, _ := a.m.db.GetDeleteCheckpointsInModelRegistry(deleteCheckpoints)
```